server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx

```diff
@@ -97,50 +98,57 @@ export default class SettingsNav extends React.PureComponent<Props> {
         <a
           className={configurationClassNames}
           data-toggle="dropdown"
-          id="settings-navigation-configuration"
-          href="#">
+          href="#"
+          id="settings-navigation-configuration">
           {translate('sidebar.project_settings')} <i className="icon-dropdown" />
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWHDO8Ba1hZr15o4gqrJ&open=AWHDO8Ba1hZr15o4gqrJ)

[ISSUE_KEY]: AWHDO8Ba1hZr15o4gqrJ